docs/How-We-Work/team-charter/our-workflow.md

```diff
@@ -1,129 +1,67 @@
+- Refined issues should be pointed, which will be based on the amount and complexity of work ([Practical Fibonacci: A Beginner's Guide to Relative Sizing](https://www.scrum.org/resources/blog/practical-fibonacci-beginners-guide-relative-sizing)), as well as other details defined in the ticket (e.g. open questions, risks, etc.). Issue points _do not_ directly correlate to hours/time spent on an issue. 
+- Pointed issues should be assigned to at least one team member before acceptance into the sprint.
+- Once the issue has been accepted into the sprint:
+    - the issue should move to the "Current Sprint" column on the Product Backlog Board and "Current Sprint Backlog" on the Sprint Board in Zenhub.
+    - the issues should be assigned to the sprint.
+    - Sprint goals will be added as a pinned post in the [TDP General Channel](https://github.com/raft-tech/TANF-app/blob/raft-tdp-main/docs/How-We-Work/team-charter/communication-tools.md#using-github-plugin-for-mattermost) at the start of each sprint. Sprint goals are also documented in each [Sprint Summary](https://github.com/raft-tech/TANF-app/tree/raft-tdp-main/docs/Sprint-Review).
+
+### Work on the issue is started
+- When the individual assigned to work on an issue starts work on it, they should move it to the “In Progress” column/pipeline in Zenhub. 
+- Progress should be updated in issue daily. Any major decisions or discussions about the issue--especially **scope creep** and **blockers** should be documented within the issue.
+- **If the issue grows in scope**:
+  - Determine with the PO, ACF Tech Lead, and PM if more refinement is needed and/or new issue(s) should be spun out from parent ticket. 
+  - Any new issues should be discussed at the next backlog refinement, unless it is a hot fix or other high priority that requires earlier discussion.
+- **If the issue becomes blocked**:
+  - Move the issue to the “Blocked” column/pipeline in Zenhub.
+  - Document (and include references and visual evidence if applicable) in the issue and indicate if the issue is _not resolvable_ or _dependent on completion of other issues_. Assignee should report on a blocked issue in standup.
+  - If dependent on other issues, [document those dependencies within the issue.](https://help.zenhub.com/support/solutions/articles/43000010349-)
+- Pairing session(s) with ACF are encouraged throughout the sprint to increase transparency and familiarity with the issue.
+
+### The issue is ready for Raft review
+- When the issue is ready for review, the assignee should check that:
+  - all ACs are met and tasks complete
+  - documentation necessary to review the work is captured
+  - A pull request is drafted using the [PR template]() for issues that will result in changes to the documentation/code managed in the TDP repo. **See [ADR 009]() for more details on the PR review process, which may vary depending on the issue type (e.g. frontend, security controls, dependencies)**.
```

Comment:
this is not pointing anywhere yet because I think ADRs should be moved to the tech doc subdir.